### ISSUE TITLE
Component: Calendar "Tab-traps" when in inline mode

### ISSUE THREAD

**Issue by jaemo** [primefaces/primevue#2640]
### Describe the bug

When using the Calendar Component in inline mode, focus is trapped within the calendar, and a user is unable to backtrack to previous inputs.

### PrimeVue version

3.13.0

### Vue version

3.x

### Language

ALL

### Build / Runtime

TypeScript

### Steps to reproduce the behavior

1) Visit the [Demo Page](https://primefaces.org/primevue/calendar)
2) Place focus on any calendar field prior to the inline option and begin tabbing.
3) Once you have reached the inline calendar, begin shift-tabbing to reverse direction of tabinput.
4) Note that you will be unable to reach the preceding inputs. This affects any inline calendar implementation with form elements that precede a calendar.

### Expected behavior

A user is able to reach form inputs that precede an inline calendar via shift-tab.

**Comment by tugcekucukoglu** [comment]
Accessibility for Calendar is under the development now. With next version, it will be fixed.

Thanks,
